internal/pkg/runtime/launcher/oci/oci_attach_linux.go

```diff
@@ -74,7 +72,18 @@ func Attach(_ context.Context, containerID string) error {
 	if err != nil {
 		return fmt.Errorf("while connecting to attach socket: %w", err)
 	}
+	// Set up context-based cancellation of conn
```

Comment:
Having thought about this, I'm not sure this one is quite right. I think once we are at readStdio we don't want a ctrl-c context cancellation from the singularity process to happen, because we are attached to the container process and should be propagating signals instead.

Could you remove this one, and I'll make a note to come back to it? Apologies.